python/feathub/processors/flink/table_builder/over_window_utils.py

```diff
@@ -239,8 +239,10 @@ def _get_over_window_agg_select_expr(
         result = expr.sum
     elif agg_func == AggFunc.VALUE_COUNTS:
         result = native_flink_expr.call("value_counts", expr)
-    # TODO: FIRST_VALUE AND LAST_VALUE is supported after PyFlink 1.16 without
-    # PyFlink UDAF.
+    elif agg_func == AggFunc.FIRST_VALUE:
```

Comment:
Should we update `FlinkProcessorITTest` to enable the corresponding tests?

Reply:
I updated the `FlinkProcessorITTest` to enable the corresponding tests.